cfgov/jinja2/v1/_includes/molecules/text-introduction.html

```diff
@@ -27,12 +27,14 @@
 <h1>{{ value.heading }}</h1>
 <div class="lead-paragraph">{{ parse_links(value.intro) | safe }}</div>
 {{ parse_links(value.body) | safe }}
-<ul class="list list__links">
-    {% for link in value.links %}
-        <li class="list_item">
-            <a class="list_link" href="{{ link.url }}">
-                {{ link.text }}
-            </a>
-        </li>
-    {% endfor %}
-</ul>
+{% if value.links.url %}
```

Comment:
Hmmm... You know, the futzing around with the item introduction, it actually seems a little off. Going to update both of these in a second, the testing instructions will be a little more involved though.